internal/cli/atlas/metrics/databases/describe.go

```diff
@@ -59,14 +59,22 @@ func (opts *DatabasesDescribeOpts) Run() error {
 	return opts.Print(r)
 }
 
-// mcli om metric(s) disk(s) describe <hostId:port> <name> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
-func DatabasesDescribeBuilder() *cobra.Command {
+// mcli atlas metric(s) database(s) describe <host:port> <databaseName> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
 	const argsN = 2
-	opts := &DatabasesDescribeOpts{}
+	opts := &DescribeOpts{}
 	cmd := &cobra.Command{
-		Use:   "describe <hostId> <name>",
+		Use:   "describe <host:port> <databaseName>",
 		Short: "Describe database measurements for a given host database.",
```

Comment:
Please change to: "Describe database metrics, such as <a, b, c>, for a database on the specified host.",